### DISCUSSION TITLE
Since new version, can't link downloads to free download manager

### DISCUSSION THREAD

**Discussion by slatermill** [aclap-dev/video-downloadhelper#799] [Help and Q&A] [answered]
### Your question

Until latest version (the new look one, not sure if there were small ones after that), every time I clicked download it automatically started the download via "Free download manager", but since then It never does. I didn't change any setting in either.

Is there a way to get it back to "the old way"?

### Addon & CoApp version and details

version: 9.0.2.3
target: google
channel: stable
lang: he
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "D:\\downloads",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "D:\\downloads"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 12,
    "successfull_dl": 2
}

**Comment by kdpainter** [comment]
Well, the new update quit working on ALL sites for me.  DRM doesn't explain it.  Not only that, I have lifetime use of faphouse and VDH doesn't work on that site but I can use the download link to download natively.
Don't be a tool.  The devs f-ed it up.

    On Monday, June 3, 2024 at 07:50:26 PM PDT, GrampaWildWilly ***@***.***> wrote:  
 
 


I do not work for VDH. I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3672, Firefox 126.0.1 64-bit, licensed VDH 9.0.2.3a1 beta, CoApp 2.0.19. VDH works just fine for me.

You folks are making empty complaints. Open PROPER problem reports & let's see if we can get solutions. In the case of Stripchat, OnlyFans, & a number of other such sites, I do believe they have implemented DRM. I'm not a member of any of these sites so my understanding is gained second-hand from posts in this forum. Digital Rights Management is a way for content creators to enforce copyright protection on their content. DRM is expressly intended to prevent downloads. It is not a bug in VDH that causes VDH to fail on this content. It is VDH operating correctly, honoring the DRM protection. If you find other tools that break the DRM protection, that's up to you & of no concern in this forum.

**Reply by GrampaWildWilly** [reply]
That's entirely untrue.  The product works fine for me.  I can't force you to trust me to find a solution for you.  But if you don't open a new discussion with a PROPER problem report, I can't help you.  Making a blanket claim that VDH fails on all sites is simply not my experience.  It's up to you whether you avail yourself of my experience.  Open a new discussion & PROPERLY report a problem or quit making noise here.

**Reply by kdpainter** [reply, marked as answer]
I am retracting everything I have said.  Turns out, the destination device was full.  They previous version gave an error that said as much.  The new version gives an unhelpful "Sorry..." message.  So, if I have a complaint, I would like a more helpful error message.  Otherwise, you are correct, it works fine.

**Comment by kdpainter** [comment]
Ok.  It doesn't work anymore where it used to.  On EVERY site.  Give me a test URL and I am sure it will fail.  How do I open a "proper" problem report when it completely fails on every site?  You are just pissing me off now.  I paid for a license and now it doesn't work.

    On Monday, June 3, 2024 at 10:31:36 PM PDT, GrampaWildWilly ***@***.***> wrote:  
 
 


That's entirely untrue. The product works fine for me. I can't force you to trust me to find a solution for you. But if you don't open a new discussion with a PROPER problem report, I can't help you. Making a blanket claim that VDH fails on all sites is simply not my experience. It's up to you whether you avail yourself of my experience. Open a new discussion & PROPERLY report a problem or quit making noise here.

**Reply by GrampaWildWilly** [reply]
Pissing you off may be loads of fun for me but it is actually not my intent.  You can believe that or not, your choice.  On every page of this forum, there is a link near the top labelled Wiki.  Click that.  Scroll the page down a bit until you see this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/3f75a77e-cfc3-476f-a2af-28f34e9d1e53)

That will take you to a page where you will find instructions for creating a proper problem report.  You have to read some documentation, click some links, read some more.  Can't be helped.  If you want to acquire new knowledge, you have to put some minimal effort into it.  You will find a page where there is a dialog you must fill out in order to create a proper problem report.  Don't skip any of the steps.  You need to tell us what software you're running & you need to tell us at least one URL of a web page where there is a video you are having problems downloading.  Don't be afraid to post more than one URL if you think that will add to the problem report.

A note about URLs.  You need to supply URLs that do not require logins.  I don't care if it might be a free registration.  I'm not signing up for some site I've never heard of just to investigate this problem.  And I especially won't sign up for a pay site.  So if you can supply some free URLs, you would be helping your own cause greatly.

As for the Wiki, it is the closest thing we have to a user manual.  You might just learn a few things if you take the time to read through all the chapters of the Wiki.  For example, the Wiki mentions beta versions of VDH.  You would probably save yourself at least one iteration of questions & answers if you installed the beta version of VDH for your browser.  The beta versions are not experimental pre-releases.  They contain bug corrections.  There's no sense in your reporting a bug that has already been solved.  Install the beta.

I feel compelled to repeat:  I do not work for VDH.  I'm just another user.  A fellow user trying to help a fellow user out of the goodness of his heart.  Be nice to me & I'll be nice to you.

**Reply by kdpainter** [reply]
See my reply above.

**Comment by GrampaWildWilly** [comment]
> I am retracting everything I have said. Turns out, the destination device was full. They previous version gave an error that said as much. The new version gives an unhelpful "Sorry..." message. So, if I have a complaint, I would like a more helpful error message. Otherwise, you are correct, it works fine.

This is excellent news @kdpainter.  Thank you for owning up to this & posting your response.  A most admirable act on your part.  I agree, a better error message does seem to be indicated.

If you encounter another issue, please open a new discussion & follow my instructions above for how to post a proper problem report.  I remain eager to help anybody who makes a proper problem report.
